flexmeasures/data/models/planning/process.py

```diff
@@ -10,30 +10,30 @@
 
 from flexmeasures.data.queries.utils import simplify_index
 from flexmeasures.data.models.time_series import Sensor
-from flexmeasures.data.schemas.scheduling.shiftable_load import (
-    ShiftableLoadFlexModelSchema,
-    LoadType,
+from flexmeasures.data.schemas.scheduling.process import (
+    ProcessSchedulerFlexModelSchema,
+    ProcessType,
     OptimizationSense,
 )
 from flexmeasures.data.schemas.scheduling import FlexContextSchema
 
 
-class ShiftableLoadScheduler(Scheduler):
+class ProcessScheduler(Scheduler):
 
     __version__ = "1"
     __author__ = "Seita"
 
     def compute(self) -> pd.Series | None:
-        """Schedule a fix load, defined as a `power` and a `duration`, within the specified time window.
+        """Schedule a prrocess, defined as a `power` and a `duration`, within the specified time window.
```

Comment:
typo: "prrocess"  is your spellchecker off? :)

Reply:
ups, thanks for noticing! :smile: